apps/web/lib/getCurrency.ts

```diff
@@ -0,0 +1,30 @@
+import { z } from "zod";
+
+import prisma from "@lib/prisma";
+
+const getCurrency = async (userId: number) => {
+  const paymentCredential = await prisma.credential.findFirst({
+    where: {
+      userId: userId,
+      type: {
+        contains: "_payment",
+      },
+    },
+    select: {
+      type: true,
+      key: true,
+    },
+  });
+
+  if (paymentCredential === null) return "usd";
+
+  if (paymentCredential.type === "stripe_payment") {
+    const stripeKeySchema = z.object({
+      default_currency: z.string(),
+    });
+    const { default_currency } = stripeKeySchema.parse(paymentCredential.key);
+    return default_currency;
+  }
```

Comment:
Created this to future proof if we have more than one payment app

Reply:
I think it’s overkill, evenTypes should have the currency already. There’s no need for an extra query just to get the currency